tests/queries/0_stateless/01158_zookeeper_log_long.sql

```diff
@@ -20,7 +22,7 @@ select 'parts';
 select type, has_watch, op_num, path, is_ephemeral, is_sequential, version, requests_size, request_idx, error, watch_type,
        watch_state, path_created, stat_version, stat_cversion, stat_dataLength, stat_numChildren
 from system.zookeeper_log
-where (session_id, xid) in (select session_id, xid from system.zookeeper_log where path='/test/01158/' || currentDatabase() || '/rmt/replicas/1/parts/all_0_0_0')
+where (session_id, xid) in (select session_id, xid from system.zookeeper_log where path='/test/01158/' || currentDatabase() || '/rmt/replicas/1/parts/all_0_0_0' and op_num not in(3))
```

Comment:
Here I excluded `Exists` query from request result because AFAIS it's done from background thread and there is no guarantee that it'll be called during. Locally, I observe that `Exists` request is part of the test output for release build and absent for debug build.

Stack example: https://pastila.nl/?016c868b/9f840f5cb1dc2400a9bd466c14340882

Reply:
> it's done from background thread and there is no guarantee that it'll be called during

Yes, you're right. We can add another `system sync replica rmt` before `system flush logs` to provide this guarantee